pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -1830,6 +1834,11 @@ type Device struct {
 	//     - name: layer2 vip example
 	//     - value: networkConfigVIPLayer2Example
 	DeviceVIPConfig *DeviceVIPConfig `yaml:"vip,omitempty"`
+	//   description: |
+	//     Picks a network device using the selector.
```

Comment:
we need to describe here or for each field how do we match on selectors: partial match, prefix match, etc.